### PR TITLE
Fix botframework send_elements

### PR DESCRIPTION
**Pull request by 1tupac**
**Proposed changes**:
I just updated my Rasa setup by bumping all my project from Core + NLU to Rasa. I noticed that attachments sent by actions through `utter_custom_message` don't work anymore. The issue I found is the double encapsulation of `elements`, leading to an issue similar to [this](https://github.com/RasaHQ/rasa/issues/3557) and also [this one](https://github.com/RasaHQ/rasa/issues/3370) I guess. 

Let me know what do you think of that quick fix. 

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by msamogh** [comment]
Thanks for the PR, we'll give it a review as soon as possible.

File: rasa/core/channels/botframework.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def prepare_message(
     async def send_elements(
         self, recipient_id: Text, elements: Iterable[Dict[Text, Any]], **kwargs: Any
     ) -> None:
-        for e in elements:
+        for e in elements[0]:
```
Comment (tmbo) [inline rasa/core/channels/botframework.py]:
```suggestion
        if not elements:
            return
        for e in elements[0]:
```

Comment (tmbo) [reply rasa/core/channels/botframework.py]: this is a bit of an odd piece of code, would be great if you can also add a reference to the botframework connector docs

**Comment by 1tupac** [comment]
Ooops mea culpa. 
Actually when I migrated from Core to Rasa 1, I didn't update my action code with `dispatcher.utter_elements` and the depreciated method `dispatcher.utter_custom_message` makes the double encapsulation of elements. Dumb fix!

**Comment by erohmensing** [comment]
Hey @1tupac ! I think I just had someone run into your same issue. The double encapsulation is my fault -- I had meant for it to just pass on the args to the `utter_elements` function but must have messed that up. Would you be up for submitting a fix in the sdk repo? 

I assume the issue is that [this line](https://github.com/RasaHQ/rasa-sdk/blob/master/rasa_sdk/executor.py#L39) should actually take `*elements` instead of `elements` no?

**Pull request closed without merging**